src/commands/stats.civet

```diff
@@ -0,0 +1,22 @@
+{ SlashCommandBuilder, ChatInputCommandInteraction, CacheType, version as djsVersion } from discord.js
+{ version as civetVersion } from ../../node_modules/@danielx/civet/package.json
+{ version as prettierVersion } from prettier/package.json
+
+data := new SlashCommandBuilder()
+		.setName "stats"
+		.setDescription "Shows the current stats of the bot."

```

Comment:
These are versions, not stats. I propose renaming the command to `versions` unless it changes to include actual statistics.